pipeline/ingestor/ingestor_works/src/test/scala/weco/pipeline/ingestor/works/CreateTestWorkDocuments.scala

```diff
@@ -0,0 +1,379 @@
+          ontologyType = "Period"
+        )
+      ),
+      label = "Past Prehistory",
+      range = None
+    )
+
+    val genre = Genre(
+      label = "Electronic books",
+      concepts = List(concept0, concept1, concept2)
+    )
+
+    saveWork(
+      work = denormalisedWork()
+        .title("A work with different concepts in the genre")
+        .genres(List(genre)),
+      description = "a work with different concepts in the genre",
+      id = "works.genres"
+    )
+  }
+
+  it("creates examples to use in the subject filter/aggregation tests") {
+    val paleoNeuroBiology = createSubjectWith(label = "paleoNeuroBiology")
+    val realAnalysis = createSubjectWith(label = "realAnalysis")
+
+    val subjectLists = List(
+      List(paleoNeuroBiology),
+      List(realAnalysis),
+      List(realAnalysis),
+      List(paleoNeuroBiology, realAnalysis),
+      List.empty
+    )
+
+    val works = subjectLists
+      .map { denormalisedWork().subjects(_) }
+
+    saveWorks(
+      works = works,
+      description = "works with different subjects",
+      id = "works.subjects"
+    )
+  }
+
+  it("creates examples to use in the contributor filter/aggregation tests") {
+    val agent47 = Agent("47")
+    val jamesBond = Agent("007")
+    val mi5 = Organisation("MI5")
+    val gchq = Organisation("GCHQ")
+
+    val agentCombos = List(
+      List(agent47),
+      List(agent47),
+      List(jamesBond, mi5),
+      List(mi5, gchq)
+    )
+
+    val works = agentCombos.map { agents =>
+      denormalisedWork().contributors(agents.map(Contributor(_, roles = Nil)))
+    }
+
+    saveWorks(
+      works = works,
+      description = "works with different contributor",
+      id = "works.contributor"
+    )
+  }
+
+  private def saveWork(
+    work: Work[WorkState.Denormalised],
+    description: String,
+    id: String
+  ): Unit =
+    saveWorks(works = List(work), description, id)
+
+  private def saveWorks(
+    works: Seq[Work[WorkState.Denormalised]],
+    description: String,
+    id: String
+  ): Unit = {
+    val documents = if (works.length == 1) {
+      val work = works.head
+
+      Seq(
+        id -> ExampleDocument(
+          description,
+          id = work.id,
+          document = work.toDocument
+        )
+      )
+    } else {
+      works.zipWithIndex
+        .map {
+          case (work, index) =>
+            s"$id.$index" -> ExampleDocument(
+              description,
+              id = work.id,
+              document = work.toDocument
+            )
+        }
+    }
+
+    saveDocuments(documents)
+  }
+
+  implicit class WorkOps(work: Work[WorkState.Denormalised]) {
+    def toDocument: Json = {
+      // This is a fixed date so we get consistent values in the indexedTime
+      // field in the generated documents.
+      val transformer = new WorkTransformer {
+        override protected def getIndexedTime: Instant =
+          Instant.parse("2020-10-15T15:51:00.00Z")
```

Comment:
Why this date?

Reply:
It's somewhat arbitrary; there's code somewhere else that hard-codes an Instant. I'll change it so it's something obviously not meaningful.